src/ZulipMobile.js

```diff
@@ -16,14 +13,6 @@ export default class ZulipMobile extends Component {
 
   componentWillMount() {
     restore(() => {
-      store.subscribe(() => {
```

Comment:
I moved this to NavigationContainer, and simplified the code.
Now NavigationContainer just gets `needsInitialFetch' as a property and it triggers the reload if needed.